testsuite/.rubocop_todo.yml

```diff
@@ -401,6 +401,11 @@ Style/DocumentationMethod:
     - 'features/support/namespaces/system.rb'
     - 'features/support/twopence_init.rb'
 
+# Offense count: 1
+Style/For:
+  Exclude:
+    - 'features/step_definitions/command_steps.rb'
+
```

Comment:
Reason why I added this - https://github.com/rubocop/ruby-style-guide#no-for-loops
I need the variable outside of the array